lib/ui/bottom_navigation/components/bottom_navigation_view.dart

```diff
@@ -46,7 +49,7 @@ class BottomNavigationView extends StatelessWidget {
                     currentIndex: state.selectedPage,
                     onTap: (int index) {
                       BlocProvider.of<BottomNavigationBloc>(context).add(BottomNavigationEvent.onPageSelected(index));
-                      if (index == 3) {
+                      if (index == (GetIt.I.get<RemoteConfigService>().isWalletEnabled ? 4 : 3)) {
```

Comment:
minor thing but I was wondering if there's not a more elegant way to map this bottom array to functions like "isSettings".

Reply:
Si, i was being lazy... Creating an enum to handle these things now :)

Reply:
check it out now. Its all controlled by the state and enums. no more mgic numbers around